tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -328,6 +328,19 @@ class CGMSHLSLRuntime : public CGHLSLRuntime {
 };
 } // namespace
 
+static uint32_t GetIntConstAttrArg(ASTContext &astContext, const Expr *expr,
```

Comment:
You've got a fair amount of mixing coding style, and DXC is a bit inconsistent.

In general if you're adding code in a part of the codebase that isn't otherwise following a consistent style we follow the [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html). LLVM uses `CamelCase` for variable names and type names (see: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly).